command.go

```diff
@@ -247,6 +247,11 @@ type Command struct {
 	// line of a command when printing help or generating docs
 	DisableFlagsInUseLine bool
 
+	// DisableVerboseFlagsInUseLine will add only [flags] to the usage line of
+	// a command when printing help or generating docs instead of a more verbose
+	// form showing the actual available flags
+	DisableVerboseFlagsInUseLine bool
```

Comment:
This is consistent with other options, but this change may visually break existing tools using this library. Should we use `EnableVerboseFlagsInUsageLine` so one can opt in to use the new feature?

Reply:
This is an important point.  Let's see where the final change lands and we can decide if we need to be less aggressive in pushing this change